### DISCUSSION TITLE
Hotkey to Switch to Next or Previous Object in Editor

### DISCUSSION THREAD

**Discussion by ChenMerami** [ppy/osu#29579] [Ideas] [answered]
Basically duplicate of this https://osu.ppy.sh/community/forums/topics/345448 but for lazer. Very useful feature for hitsounding a map using only hotkeys, would increase speed drastically as you could do everything using keyboard only (by just pressing a key instead of navigating with mouse)

**Comment by Sanch-KK** [comment]
agree as a mapper, very useful

**Comment by Tenexxt** [comment]
Personally I'd prefer for it to be set on the scroll, but I guess I could just bind it

**Comment by bdach** [comment, marked as answer]
As I said in that thread something like this is already in the works. https://github.com/ppy/osu/pull/28737
